README.md

```diff
@@ -27,3 +28,9 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+#Answer
+I think a good invariant would be the array that holds the numbers in the sequence. I think to generalize I would call it arr[(i-2),(i-1)]. 
```

Comment:
So what specifically would you say has to be true at the beginning of each iteration?